pages/test_analytics/swift_collectors.md.erb

```diff
@@ -12,9 +12,17 @@ You can also upload test results by importing [JSON](/docs/test-analytics/import
 
 Before you start, make sure XCTest runs with access to [CI environment variables](/docs/test-analytics/ci-environments).
 
-1. [Create a test suite](https://buildkite.com/docs/test-analytics), and copy the _Buildkite Test Analytics API token_ you see
+1. [Create a test suite](https://buildkite.com/docs/test-analytics) and copy the test suite API token.
 
-1. Add the `test-collector-swift` repository to the `Package.swift` manifest, and add `BuildkiteTestCollector` to any test target requiring analytics:
+1. [Securely](/docs/pipelines/secrets) set the `BUILDKITE_ANALYTICS_TOKEN` secret on your CI to the API token from the previous step.
+
+   If you're testing an Xcode project, note that Xcode doesn't automatically pass environment variables to the test runner, so you need to add them manually.
+   In your test scheme or test plan, go to the _Environment Variables_ section and add the following key-value pair:
```

Comment:
Unless I'm mistaken, this isn't the Buildkite environment variables section, which is why I didn't capitalize it first time round. We could just say it looks better like this. :-p 

See also https://github.com/buildkite/docs/pull/1661#discussion_r928943201

Reply:
I dug into this a little more. I think this is referring to an Xcode UI string, which is capitalized.